runtime/jvmti/jvmtiCapability.c

```diff
@@ -372,6 +371,19 @@ jvmtiAddCapabilities(jvmtiEnv* env,
 			}
 		}
 
+#if JAVA_SPEC_VERSION >= 11
+		if (1 == newCapabilities.can_generate_sampled_object_alloc_events) {
```

Comment:
I'd prefer not to see the `1 ==`. These bits are booleans.

Reply:
It is a `unsigned int` instead of `boolean` as per https://github.com/eclipse/openj9/blob/ba27f519d48a2a4b6dd811212ec106313b734052/runtime/include/jvmti.h.m4#L487

Reply:
It's a single bit inside an unsigned int, actually, so it will only ever be 0 or 1.

Reply:
Agreed, updated the PR accordingly, please have another look.